Backend/app/models/user.rb

```diff
@@ -9,7 +11,27 @@ class User < ApplicationRecord
    has_many :room_bookings
    has_many :event_bookings
 
-   enum role: ["admin","professor","student"]
```

Comment:
I am surprised rubocop is not failing on this. there should be a space after each comma